### PR TITLE
Added new description to troubleshooting.md

### PR DESCRIPTION
**Pull request by MarcusSchoelzel**
Follow up to a Slack post: https://stryker-mutator.slack.com/archives/CTYUW4B0E/p1634123487026100.
Sorry that it took that long.

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -205,5 +205,23 @@ Override the default babel plugins to solve this issue. For example, leaving it
...
+The mutated source file is only implementing callbacks for a framework that uses express in the background as a server. The unit tests for this module are written with Jest and are using Axios to make REST calls against the server. There is no module dependency needed for the test. But Stryker is using Jest's `--findRelatedTests` to optimize the performance. And it looks like that this Jest feature is only looking at imports (see this [StackOverflow post](https://stackoverflow.com/questions/44066996/how-does-jest-findrelatedtests-work-under-the-hood).
+
+**Solution**
+
+Adding an (unused) import for the tested module get the mutants killed. This is not an optimal solution but it works.
+
+Example:
+
+```typescript
+import {} from "../FileUnderTest";
```
Comment (nicojs) [inline docs/troubleshooting.md:226]: I think `import '../FileUnderTest'` would work as well, right?

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -205,5 +205,23 @@ Override the default babel plugins to solve this issue. For example, leaving it
...
+
+For a TypeScript file the mutants don't get killed but a coverage (in Jest) is shown.
+
+**Problem**
+
+The mutated source file is only implementing callbacks for a framework that uses express in the background as a server. The unit tests for this module are written with Jest and are using Axios to make REST calls against the server. There is no module dependency needed for the test. But Stryker is using Jest's `--findRelatedTests` to optimize the performance. And it looks like that this Jest feature is only looking at imports (see this [StackOverflow post](https://stackoverflow.com/questions/44066996/how-does-jest-findrelatedtests-work-under-the-hood).
+
+**Solution**
+
+Adding an (unused) import for the tested module get the mutants killed. This is not an optimal solution but it works.
```
Comment (nicojs) [inline docs/troubleshooting.md:221]: Maybe we can also mention that `--findRelatedTests` can also be disabled? https://stryker-mutator.io/docs/stryker-js/jest-runner#jestenablefindrelatedtests-boolean

File: docs/troubleshooting.md (hunk 1 above)
Comment (MarcusSchoelzel) [reply docs/troubleshooting.md:226]: Yes, it depends on the preference of the developer/project. Should i change it?

**Review by nicojs — APPROVED** [review]

**Comment by nicojs** [comment]
Thanks @MarcusSchoelzel

File: docs/troubleshooting.md (hunk 1 above)
Comment (nicojs) [reply docs/troubleshooting.md]: Sure, we can leave it like this.

**Pull request merged**